api/__init__.py

```diff
@@ -22,6 +27,29 @@
 app = Flask(__name__)
 
 
+# Asset file types
+file_types = [
+    ('text', ['.vtt', '.txt', '.srt', '.json']),
+    ('markup', ['.xml']),
+    ('image', ['.png', '.jpeg', '.jpg']),
+    ('audio', ['.mp3', '.wav']),
+    ('video', ['.mp4', '.mov', 'webm', 'mkv'])]
+
```

Comment:
I don't like these changes here, addressing discussion in #1 . I'm not a fan of a big conglomerate branch that tries to fix everything, and there must be a separate PR that addresses that issue alone.

Reply:
Reverted this and will eventually add it to a new branch.